stratum/hal/lib/common/common.proto

```diff
@@ -64,14 +64,14 @@ enum TrafficClass {
 };
 
 // Mode of hotplug request
-enum SWBackendQemuHotplugStatus {
-  NO_HOTPLUG = 0;
-  HOTPLUG_ADD = 1;
-  HOTPLUG_DEL = 2;
+enum QemuHotplugMode {
```

Comment:
I'm not convinced that this parameter is idiomatically correct. It's a common, not a mode or state. A more conventional interface might be a read/write Boolean `hotplug-enabled` (true or false) plus a read-only `hotplug-state` (e.g INACTIVE, CONFIGURING, ACTIVE, and one or two error states).